docs/control-structures.rst

```diff
@@ -29,8 +29,13 @@ something like::
         }
     }
 
-Input parameters can be used just as any other local variable
-can be used, they can also be assigned to.
+Input parameters can be used as any other local variable and they can also be assigned to.
+
+.. note::
+    A function cannot accept a multi-dimensional array as an input parameter.
```

Comment:
This is missing one important keyword which was present in the FAQ: "non-internal function".

Basically a function marked internal supports it without any tweaks.  Functions which are external or public only support it if ABIEncoderV2 is enabled.